frontend/config.json

```diff
@@ -10,12 +10,14 @@
     "h2": "",
     "h3": "",
     "background": "",
-    "textColor": ""
```

Comment:
Why is the config.json file committed to the repository? I think this file should be ignored by git.

Reply:
i think this file was added to git before someone put it to the git ignore. will delete it